static/js/util/util.js

```diff
@@ -227,44 +228,60 @@ complete profile consists of:
     `currently employed == 'yes'`
 */
 export function validateProfileComplete(profile) {
+  let errors = {};
   let reqFields = [];
 
   // check personal tab
-  reqFields = reqFields.concat(...PersonalTab.defaultProps.requiredFields);
-  let errors = validateProfile(profile, reqFields, {});
-  if ( !_.isEqual(errors, {}) ) {
-    return ([false, {
-      url: "/profile/personal",
-      title: "Personal Info",
-      text: "Please complete your personal information.",
-    }]);
+  errors = validateProfile(
+    profile,
+    PersonalTab.defaultProps.requiredFields,
+    PersonalTab.defaultProps.validationMessages
+  );
+
+  if (!_.isEqual(errors, {})) {
+    return [false, '/profile/personal', errors];
   }
 
   // check professional tab
-  if ( _.isArray(profile.work_history) && !_.isEmpty(profile.work_history) ) {
+  if ( _.isArray(profile.work_history) && !_.isEmpty(profile.work_history)) {
```

Comment:
total style nit: for `if .. else` conditions I think it's best to do one of the following:

```js
if ( condition && otherCondition ) {
  ...
}
```
or 
```js
if (condition && otherCondition) {
  ...
}
```

just consistency in the spacing of the wrapping parentheses.